main.tf

```
@@ -71,7 +71,7 @@ module "main" {
     },
     {
       name  = "AUTH_APP_URL"
-      value = "https://${var.env_name}.${local.domain[var.fp_context]}"
+      value = local.auth_app_url[var.fp_context]}
```

```suggestion
      value = local.auth_app_url[var.fp_context]
```
I think deploy_review failed because of the extract `}`.

Otherwise looks good
Did deploy_build